frontend/src/components/NavBar.jsx

```diff
@@ -0,0 +1,38 @@
+// NavBar.js
+
+import React from 'react';
+import { Link, useMatch, useResolvedPath } from 'react-router-dom';
+import './NavBar.css';
+
+const NavBar = ({ navigate }) => {
+    const logout = () => {
+      window.localStorage.clear();
+      window.localStorage.removeItem('token');
+      window.localStorage.removeItem('token');
```

Comment:
There are 3 lines here that seem to perform the same purpose. I am assuming that clear() will remove everything including the token.